app/views/rooms/index.html.erb

```diff
@@ -69,6 +69,10 @@
         <input type="checkbox" <%= room.display_on_home_page ? "checked" : "" %> disabled>
       <% end %>
 
+      <% table.with_column(Room.human_attribute_name(:status), sort_by: :status) do |room| %>
+        <%= Room.human_attribute_name("status.#{room.status}") %>
+      <% end %>
```

Comment:
Maybe we could use a badge or label? And use decorator to apply color type in function of value

Reply:
ok